bandit/blacklists/calls.py

```diff
@@ -265,20 +265,6 @@
 | B321 | ftplib              | - ftplib.\*                        | High      |
 +------+---------------------+------------------------------------+-----------+
 
-B322: input
```

Comment:
@ericwb removed B322 inline with https://github.com/PyCQA/bandit/issues/596

Was not sure if I should leave B322 as a gap or I should count -1 on the other remain tests. So for example B323 becomes B322, B324 becomes B323 etc. I was thinking around folks who are cherry picking tests to run.

Reply:
Yes, we have to keep legacy numbers for historical reasons as to not break users including/excluding certain numbers.  We have other examples where we leave the documentation for the ID, but indicate to the users that the test has been removed.